grape.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'activesupport'
   s.add_runtime_dependency 'builder'
-  s.add_runtime_dependency 'dry-types', '~> 1.1.1'
+  s.add_runtime_dependency 'dry-types', '~> 1.1'
```

Comment:
Do we need to lock it at all? I think unless there's a specific reason that version x.y.z doesn't work we should leave this blank.

Reply:
Actually  there were cases with dry-types when it has some incompatibility issues between versions, at least it was like this before 1.0 release. As for now it must be stable enough. But if somebody has an old version - Grape just wouldn't work for him. 

I tested it now with 1.0.0 and 3 specs are failed. The same is for 1.0.1. But everything is OK starting from 1.1.0

Reply:
Should this be >= 1.1 then?

Reply:
Only if we don't afraid of breaking changes in 2.0. If not - I'll change :)

Reply:
Not afraid, let's make it `>= 1.1`, again I generally am of the opinion we should only be restricting for known issues. In this case you've identified that < 1.1 doesn't work, so it makes perfect sense. But we don't need to be afraid of future incompatibility and I would rather find sooner than later by a breaking build.

Reply:
Makes sense for me :+1: 
Done